### DISCUSSION TITLE
Using only the CLI for using DIA-NN

### DISCUSSION THREAD

**Discussion by tannishthadas** [vdemichev/DiaNN#928] [Q&A]
I was wondering if someone can share a tutorial for using DIA-NN using only the CLI without the need for GUI. I have seen the help given on the GitHub repository but am still facing a bit of confusion. If you wanted I could share the exact thing I am trying to do. Please help!

**Comment by vdemichev** [comment]
It's quite simple: can run in the GUI and then copy (from the top of the log) the commands to use in the CLI - this way can see which GUI settings correspond to which commands. The GUI also works on Linux under Wine if necessary.
